scalafix-tests/shared/src/main/scala/test/PrettyTest.scala

```diff
@@ -93,7 +93,6 @@ object Test {
     val annType1: T @ann(42) = ???
     val annType2: T @ann1 @ann2 = ???
 
-    val existentialType1: T forSome { type T } = ???
```

Comment:
Existential types are longer supported in Scala 3 and rarely used in Scala 2 so for the sake of simplicity, I dropped this completely.